Scruffy.Commands/SlashCommands/InformationSlashCommandModule.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Discord;
+using Discord.Interactions;
+using Scruffy.Services.Discord;
+using Scruffy.Services.Statistics;
+
+namespace Scruffy.Commands.SlashCommands;
+
+/// <summary>
+/// Shows general informations about Scruffy
+/// </summary>
+public class InformationSlashCommandModule : SlashCommandModuleBase
+{
+    #region Properties
+
+    /// <summary>
+    /// Visualizer
+    /// </summary>
+    public StatisticsVisualizerService VisualizerService { get; set; }
+
+    #endregion // Properties
+
+    #region Methods
+
+    /// <summary>
+    /// Shows general informations about Scruffy
+    /// </summary>
+    /// <returns>A <see cref="Task"/> representing the asynchronous operation</returns>
+    [SlashCommand("info", "Display infos about Scruffy")]
+    public async Task Info()
+    {
+        var builder = new EmbedBuilder()
```

Comment:
I would recommend to move all the logic which is not slash command specific to a command handler class.

Example:
[RaidSlashCommandModule](https://github.com/thoenissen/Scruffy/blob/main/Scruffy.Commands/SlashCommands/RaidSlashCommandModule.cs)
[RaidCommandHandler](https://github.com/thoenissen/Scruffy/blob/main/Scruffy.Services/Raid/RaidCommandHandler.cs)

Then you can use the command handler in the slash command and the text command. Then you only need to move the [translation](https://github.com/thoenissen/Scruffy/blob/main/Scruffy.Data/Resources/Languages/de-DE.json) to the corresponding command handler group.

Reply:
Done

Reply:
You need to change the group name in the [de-DE.json](https://github.com/thoenissen/Scruffy/blob/main/Scruffy.Data/Resources/Languages/de-DE.json).